vic/drivers/image/src/get_global_param.c

```diff
@@ -621,40 +621,70 @@ get_global_param(FILE *gp)
             }
-            else if (strcasecmp("GLOBAL_LAI", optstr) == 0) {
-                log_warn("GLOBAL_LAI has been replaced by 2 new options: "
-                         "VEGPARAM_LAI (whether the vegparam file "
-                         "contains LAI values) and LAI_SRC (where to get LAI "
-                         "values).");
-                log_warn("\"GLOBAL_LAI  TRUE\" should now be: \"VEGPARAM_LAI "
-                         "TRUE\" and \"LAI_SRC  LAI_FROM_VEGPARAM\".");
-                log_warn("\"GLOBAL_LAI  FALSE\" should now be: \"LAI_SRC "
-                         "LAI_FROM_VEGLIB\".");
+            else if (strcasecmp("VEGPARAM_LAI", optstr) == 0) {
                 sscanf(cmdstr, "%*s %s", flgstr);
                 if (strcasecmp("TRUE", flgstr) == 0) {
                     options.VEGPARAM_LAI = true;
-                    options.LAI_SRC = LAI_FROM_VEGPARAM;
                 }
                 else {
-                    options.LAI_SRC = LAI_FROM_VEGLIB;
+                    options.VEGPARAM_LAI = false;
                 }
             }
-            else if (strcasecmp("VEGPARAM_LAI", optstr) == 0) {
+            else if (strcasecmp("LAI_SRC", optstr) == 0) {
+                sscanf(cmdstr, "%*s %s", flgstr);
+                if (strcasecmp("FROM_VEGHIST", flgstr) == 0) {
+                    options.LAI_SRC = FROM_VEGHIST;
+                }
+                else if (strcasecmp("FROM_VEGPARAM", flgstr) == 0) {
+                    options.LAI_SRC = FROM_VEGPARAM;
+                }
+                else {
+                    options.LAI_SRC = FROM_VEGLIB;
+                }
+            }
+            else if (strcasecmp("VEGPARAM_VEGCOVER", optstr) == 0) {
                 sscanf(cmdstr, "%*s %s", flgstr);
                 if (strcasecmp("TRUE", flgstr) == 0) {
-                    options.VEGPARAM_LAI = true;
+                    options.VEGPARAM_VEGCOVER = true;
                 }
                 else {
-                    options.VEGPARAM_LAI = false;
+                    options.VEGPARAM_VEGCOVER = false;
                 }
             }
-            else if (strcasecmp("LAI_SRC", optstr) == 0) {
+            else if (strcasecmp("VEGCOVER_SRC", optstr) == 0) {
                 sscanf(cmdstr, "%*s %s", flgstr);
-                if (strcasecmp("LAI_FROM_VEGPARAM", flgstr) == 0) {
-                    options.LAI_SRC = LAI_FROM_VEGPARAM;
+                if (strcasecmp("FROM_VEGHIST", flgstr) == 0) {
+                    options.VEGCOVER_SRC = FROM_VEGHIST;
+                }
+                else if (strcasecmp("FROM_VEGPARAM", flgstr) == 0) {
+                    options.VEGCOVER_SRC = FROM_VEGPARAM;
+                }
+                else if (strcasecmp("FROM_VEGLIB", flgstr) == 0) {
+                    options.VEGCOVER_SRC = FROM_VEGLIB;
                 }
                 else {
-                    options.LAI_SRC = LAI_FROM_VEGLIB;
+                    options.VEGCOVER_SRC = FROM_DEFAULT;
```

Comment:
It would be better to fail if the `VEGCOVER_SRC` option is not recognized.  The default should be `FROM_DEFAULT` as you have it but if someone has this line in their param file, they will get the default value, which is not what they wanted:  
```
VEGCOVER_SRC LAI_FROM_VEGPARMA   # typo!
```

So you just need to put one more `else if` statment in there for `FROM_DEFAULT`, then fail if you get to the else.